code/gits_clone.py

```diff
@@ -0,0 +1,21 @@
+from subprocess import Popen, PIPE
+
+def gits_clone_func(args):
+    """
+    Function to target an existing repository and create a clone, or copy of the target repository.
+    """
+    try:
+        subprocess_command = list()
+        subprocess_command.append("git")
+        subprocess_command.append("clone")
+        site_url = args.site_url
+
+        if len(site_url) == 0:
```

Comment:
If the site url is blank, a message can be displayed to the user

Reply:
I think the user will think of it as an error if a message is printed.
Let's leave it blank that git clone <empty> means cloning nothing anyway.